articles/access_controls.md

```diff
@@ -1,58 +1,57 @@
 
-### Data that does not require Conditions for Use
-
-Open Data in Synapse are data without `Conditions for Use.` They can be used by any registered Synapse users authorized to view the project/folder/file/table (based on sharing setting) for research purpose. Typically Open Data are:
+{% include important.html content="`Controlled Data` may not be redistributed." %}
+`Controlled Data` can only be downloaded and used by authorized Synapse users. `Controlled Data` is not transferable unless explicitly specified otherwise. In other words, you cannot share `Controlled Data` with collaborators. Each Synapse user wishing to access `Controlled Data` must individually agree to the `Conditions for Use` to access that data.
 
-* Data from model organism, species or strain
-* Non-biological data such as presentations, results or images
+{% include important.html content="Synapse users are responsible for determining the appropriate `Conditions for Use` for any data they upload into Synapse." %}
+It is important to note that `Conditions for Use` cannot be set for Synapse `Wikis` or `Discussion Forums`: they are not designed to house data and therefore do not have local `Conditions for Use` as a feature. Synapse `Wikis` or `Discussion Forums` inherit the `Conditions for Use` of their Synapse parent `Project`.
 
+### Data that does not require Conditions for Use
+Synapse `Open Data` is data that does not require `Conditions for Use`. Any registered Synapse user included in the `Project`, `Folder`, `File` or `Table` (based on the `Sharing setting`) can immediately access `Open Data`. Typically, `Open Data` is:
+* Data from model organisms, species or strains
+* Non-biological data, like that used for the calibration of instruments 
 Human data that are:
 * Publicly available elsewhere
-* Anonymous data
-* De-identified non-sensitive data with no known sharing or use restrictions
-* Self-contributed data unambiguously consented for open data sharing and use
+* Anonymous
+* De-identified and non-sensitive, with no known sharing or use restrictions
+* Self-contributed and unambiguously consented for open data sharing and use
 
-### Data that does require Conditions for Use (“Controlled Use” data)
-`Conditions for Use` are typically applied in order to comply with the terms under which the data were collected or with other human subjects regulations. In Synapse, data that carries `Conditions for Use` is labeled as **Controlled Use**. Examples of Controlled Use data include:
+### Data that does require Conditions for Use
+`Conditions for Use` regulate who can access `Controlled Data` and how it can be used. This means that any Synapse user included in a `Project`, `Folder`, `File` or `Table` (based on the `Sharing setting`) must fulfill the `Conditions for Use` **before** he/she can access that `Controlled Use` data. Examples of `Controlled Data` include:
```

Comment:
Controlled Data should not be in back ticks it is not a Synapse concept "Controlled Use" is though.